test/jwks.spec.js

```diff
@@ -1,235 +0,0 @@
-'use strict'
```

Comment:
why was this file removed?

Reply:
Why was this removed?

Reply:
it was just renamed, and broken up into multiple test files